packages/frontend/src/providers/ExplorerProvider.tsx

```diff
@@ -721,9 +718,6 @@ function reducer(
                 ...state.unsavedChartVersion.metricQuery.tableCalculations.map(
                     (tc) => tc.name,
                 ),
-                ...(state.unsavedChartVersion.metricQuery.customDimensions?.map(
```

Comment:
Adding customDimensions to the list of activeFields is not redundant, they are in `dimensions` . This was not causing any issues or werid behaviours, but I know this is not needed. I'm wondering if there are any other places we need to update because of this...

Reply:
Maybe, in table viz the custom dimension is duplicated. 

<img width="1332" alt="Screenshot 2024-05-16 at 10 22 01" src="https://github.com/lightdash/lightdash/assets/9117144/7601c0bb-eb93-4d49-8a37-754c6866c515">

Reply:
Fixed table viz 
![Screenshot from 2024-05-16 13-19-55](https://github.com/lightdash/lightdash/assets/1983672/14b6ae5f-4e4e-4a0b-98b5-b415b34a6165)